docs/team/tristyxxnana.adoc

```diff
@@ -0,0 +1,86 @@
+= Nur Tristy Erna - Project Portfolio
+:site-section: AboutUs
+:imagesDir: ../images
+:stylesDir: ../stylesheets
+
+== PROJECT: Trajectory
+
+---
+
+== Overview
+
+Trajectory is a lightweight management system for those who need a no-frills solution for managing students, modules, grades, classes and notes. It is built for users who prefer typing (CLI) over GUI interaction and has a GUI created with JavaFX.
+
+== Summary of contributions
+
+* *Major enhancement*: added *the ability to display a graph command for student grades*
```

Comment:
Should be `Major feature implemented` instead of `Major enhancement`. You should also add `Code Contributed` above this line (refer to the website for details)